definitions/features/downstream.rb

```diff
@@ -1,4 +1,7 @@
 class Features::Downstream < ForemanMaintain::Feature
+  MIN_MEM = 7_900_000
+  DISPLAY_MEM = 8
```

Comment:
It would be good if we had just one value for specifying the minum memory, and the display one would be just calculated from there. If the concern is that we want to be a bit tolerant and not fail, if the real memory is 7.9GB when we require 8GB sharp, it could be handled with incorporating some tolerance (ether in absolute value of let's say 0.5GB, or 10% or something like that).

Also, please avoid using abbreviations for readability purposes, especially for variables with larger scope, so `MIN_MEMORY` would work better here.

Reply:
Thank you! I am working on changes and will send those again.